.env

```diff
@@ -2,6 +2,7 @@
 LANGUAGES_TO_RENDER=en-US
 
 # US English book paths
+DATE_FORMAT_en_US='%m/%d/%Y'
```

Comment:
Technically this has nothing to do with the fonts; it is related to fixing the date of last changes displayed in the rendered files.